renderdoc/driver/d3d11/d3d11_common.h

```diff
@@ -195,7 +195,17 @@ inline const char *GetDebugName(T *pObj)
   {
     HRESULT hr = pObj->GetPrivateData(WKPDID_D3DDebugObjectName, &size, tmpBuf);
     if(FAILED(hr))
-      return "";
+    {
+      static wchar_t tmpBufW[1024] = {0};
+      UINT wsize = size * 2;
+      hr = pObj->GetPrivateData(WKPDID_D3DDebugObjectNameW, &wsize, tmpBufW);
+      if(FAILED(hr))
+        return "";
+
+      size = wsize / 2;
+      size_t sizeConverted;
+      wcstombs_s(&sizeConverted, tmpBuf, tmpBufW, size);
```

Comment:
I have `StringFormat::Wide2UTF8` for this kind of conversion, please use that instead since these functions can cause headaches on other compilers than MSVC (which doesn't happen right now but could in future).

Since this function is only called in one place that immediately converts to `std::string` you can make life easier here by changing this function to return `std::string` and remove the need for the static arrays and any string copies after conversion.